src/box/box.cc

```diff
@@ -89,6 +89,7 @@
 #include "mp_uuid.h"
```

Comment:
> vinyl: added box API
> function for getting bloom
> size memory used by space
>
> NO_CHANGELOG=internal
> NO_DOC=internal
> NO_TEST=internal

Used by index? Please format your sentence and commit subject properly. The right subsystem for this change should be box.

Reply:
Am I right that you mean _net.box_? I don't see _box_ subsystem in the [guide](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message).

Reply:
Created an updating issue. Changed to _box_ subsystem. Done.